main/src/cgeo/geocaching/maps/MapOptions.java

```diff
@@ -38,6 +38,9 @@ public MapOptions(final Context context, @Nullable final Bundle extras) {
             waypointType = (WaypointType) extras.get(Intents.EXTRA_WPTTYPE);
             mapState = extras.getParcelable(Intents.EXTRA_MAPSTATE);
             title = extras.getString(Intents.EXTRA_TITLE);
+            if (null != coords && null == waypointType) {
```

Comment:
Under what conditions might this happen?

Reply:
This is the $1,000 question (and the reason why I called this a mitigation).
When I debug backwards from the error position I can see that this is the condition which leads to the error. I only do not know where it origins from. Also I cannot reproduce it. So I decided to go for a mitigation for now.

Reply:
maybe imported waypoints from a GPX?

Reply:
But you have to consider, that this is a new crash with current beta, which makes my previous comment unlikely.

Reply:
May be. Do you happen to have some test data for this case?

Reply:
No, it was just a guess.